build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/DownloadUri.cs

```diff
@@ -58,12 +58,22 @@ public override bool Execute ()
 				Log.LogMessage (MessageImportance.Normal, $"Skipping uri '{uri}' as destination file already exists '{destinationFile}'.");
 				return;
 			}
-			Log.LogMessage (MessageImportance.Low, $"Downloading '{uri}'.");
+			var dp  = Path.GetDirectoryName (destinationFile);
+			var dn  = Path.GetFileName (destinationFile);
+			if (string.IsNullOrEmpty (dn)) {
```

Comment:
I did, but now I don't recall what I was thinking... The check doesn't make sense.